workflow-templates/1gp-ci.yml

```diff
@@ -56,30 +56,64 @@ on:
 
       auto-promote:
         description: Whether to auto promote the package
-        value: "false"
+        value: 'false'
 
 permissions:
   id-token: write # This is required for requesting the ID token for AWS actions
   contents: read # This is required for actions/checkout
 
 concurrency:
-  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
   cancel-in-progress: ${{ github.ref != 'refs/heads/release' }}
 
 env:
-  DD_AGENT_MAJOR_VERSION: "7"
-  DD_API_KEY: "${{ secrets.DD_API_KEY }}"
-  DD_APP_KEY: "${{ secrets.DD_APP_KEY }}"
+  DD_AGENT_MAJOR_VERSION: '7'
+  DD_API_KEY: '${{ secrets.DD_API_KEY }}'
+  DD_APP_KEY: '${{ secrets.DD_APP_KEY }}'
   DD_SITE: datadoghq.com
-  DEVOPS_SCRIPTS_PATH: "./devops-utility-belt/scripts"
+  DEVOPS_SCRIPTS_PATH: './devops-utility-belt/scripts'
   GIT_BASE_BRANCH: ${{ github.base_ref }}
   GIT_BRANCH: ${{ github.ref_name }}
-  GITHUB_TOKEN: "${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}"
+  GITHUB_TOKEN: '${{ secrets.CI_USER_REPO_ACCESS_TOKEN }}'
   RELEASE_FULLNAME: nCino Companies House (UK)
   REPO_NAME: ${{ github.event.repository.name }}
+  NAMESPACE: << REPLACE WITH PROJECT NAMESPACE >>
 
 jobs:
+  Path-Filtering:
+    outputs:
+      apex_changed:: ${{ steps.path-filtering.outputs.apex_changed }}
+      lwc_changed: ${{ steps.path-filtering.outputs.apex_changed }}
+      markdown_changed:: ${{ steps.path-filtering.outputs.markdown_changed }}
+      non_markdown_changed:: ${{ steps.path-filtering.outputs.non-markdown_changed }}
```

Comment:
Nit, Kebab case would be nice here 🤔

Reply:
github's examples for outputs uses underscores so that's what I was going off of, but I agree kebab case is nicer in this context